.github/workflows/build-lint-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Build, Lint, and Test
+
+on:
+  workflow_call:
+
+jobs:
+  prepare:
+    name: Prepare
```

Comment:
This job is not strictly required here, but could be useful to have for adding other steps in the future (e.g., tests).